app/questionnaire/value_source_resolver.py

```diff
@@ -34,8 +34,8 @@ class ValueSourceResolver:
     schema: QuestionnaireSchema
     location: LocationType | None
     list_item_id: str | None
-    progress_store: ProgressStore
-    supplementary_data_store: SupplementaryDataStore
+    progress_store: progress_store.ProgressStore
+    supplementary_data_store: supplementary_data_store.SupplementaryDataStore
```

Comment:
Was some of this refactor done with a tool? Some of the changes look a bit odd

Reply:
This one is done to avoid circular import error.